bigchaindb/commands/bigchaindb.py

```diff
@@ -280,6 +281,16 @@ def run_start(args):
     start(args)
 
 
+def run_tendermint_version(args):
+    """Show the supported Tendermint version(s)"""
+    supported_tm_ver = {
+        "description": "BigchainDB supports the following Tendermint version(s)",
+        "tendermint": [],
+    }
+    [supported_tm_ver["tendermint"].append(ver) for ver in __tm_supported_versions__]
```

Comment:
It may be slightly more concise:
```
supported_tm_ver = {
    'description': 'BigchainDB supports the following Tendermint version(s)',
    'tendermint': __tm_supported_versions__
}
```
On a side note, PEP8 recommends to stick to a single type of quotes, and we generally use `'`.